### ISSUE TITLE
Transform_Graph feed value for  bool placeholder

### ISSUE THREAD

**Issue by JiahaoYao** [tensorflow/tensorflow#19683]
### System Information

- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow):** No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04):** MacOS 10.13.3
- **TensorFlow installed from (source or binary):** source
- **TensorFlow version (use command below):** r1.8
- **Python version:** 2.7
- **Bazel version (if compiling from source):** 0.13
- **GCC/Compiler version (if compiling from source):** Apple LLVM version 9.1.0 (clang-902.0.39.1)
- **CUDA/cuDNN version:** version 8.0
- **GPU model and memory:** No
- **Exact command to reproduce:** `bazel build tensorflow/tools/graph_transforms:transform_graph`

### Describe the problem
Fold the batchnorm of [Facenet](https://github.com/davidsandberg/facenet#Pre-trained).

### Source Code/ Logs
```bash
bazel-bin/tensorflow/tools/graph_transforms/transform_graph 
--in_graph="/Users/kit/Downloads/20180408-1029002/20180408-102900.pb"
 --out_graph="../optimize_graph.pb" 
--inputs="input" 
--outputs="InceptionResnetV1/Logits/Flatten/flatten/Reshape"
 --transforms='
  strip_unused_nodes(type=float, shape="1,299,299,3")
  remove_nodes(op=Identity, op=CheckNumerics)
  fold_constants(ignore_errors=true)
  fold_batch_norms
  fold_old_batch_norms'
```
### Error:
```bash
2018-06-01 12:51:11.118630: I tensorflow/tools/graph_transforms/transform_graph.cc:264] Applying strip_unused_nodes
2018-06-01 12:51:11.344986: I tensorflow/tools/graph_transforms/transform_graph.cc:264] Applying remove_nodes
2018-06-01 12:52:04.712395: I tensorflow/tools/graph_transforms/transform_graph.cc:264] Applying fold_constants
2018-06-01 12:52:05.068919: E tensorflow/tools/graph_transforms/transform_graph.cc:279] fold_constants: Ignoring error You must feed a value for placeholder tensor 'phase_train' with dtype bool
	 [[Node: phase_train = Placeholder[dtype=DT_BOOL, shape=<unknown>]()]]
2018-06-01 12:52:05.201658: I tensorflow/tools/graph_transforms/transform_graph.cc:264] Applying fold_batch_norms
2018-06-01 12:52:05.509836: I tensorflow/tools/graph_transforms/transform_graph.cc:264] Applying fold_old_batch_norms
```
I use the tensorboard to see the graph and I `phase_train` is the input to every `slim.batchnorm`.
![image](https://user-images.githubusercontent.com/20907377/40823628-9adb611c-65a3-11e8-8b0a-1f6414718460.png)

My Question is how can I feed the value for this bool placeholder in bazel.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jart: It has been 16 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @jart: It has been 31 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by qiaohaijun** [comment]
same issue
```
# transform_graph  --in_graph='freezed_graph.pb' --outputs='cond_2/final_score' --out_graph='test.pb' --transforms='fold_constants(ignore_errors=true)'
2018-08-30 15:56:35.647989: I tensorflow/tools/graph_transforms/transform_graph.cc:264] Applying fold_constants
2018-08-30 15:56:36.710932: E tensorflow/tools/graph_transforms/transform_graph.cc:279] fold_constants: Ignoring error You must feed a value for placeholder tensor 'train_flag' with dtype bool
	 [[Node: train_flag = Placeholder[dtype=DT_BOOL, shape=[]]()]]
```
